website/docs/components/form/masked-input/partials/guidelines/overview.md

```diff
@@ -0,0 +1,10 @@
+A masked input field is a special input that visually obfuscates characters to protect sensitive information by masking them with a circular shape.
+
+!!! Warning
+
+**Important to know**
+
+This component it's meant to be used for **visual obfuscation** only. Consumers should be aware that the hidden text value could still be obtained with other means (eg. copying it from the input, via JavaScript, via the developer tools, etc.).
```

Comment:
"through/by other means" would probably be better here

```suggestion
This component it's meant to be used for **visual obfuscation** only. Consumers should be aware that the hidden text value could still be obtained through other means (eg. copying it from the input, via JavaScript, via the developer tools, etc.).
```

Reply:
There are other errors so I added the below suggested edits